### PR TITLE
Add details in the readme and index.html for Changelog Nightly

### PR DESCRIPTION
**Pull request by adamstac**
Hey @igrigorik,

Here's a PR for you to review to update the readme and index.html to add the details for [Changelog Nightly](http://thechangelog.com/nightly/) and [episode #144](http://thechangelog.com/144/).

I'd love to help out with some design sometime too on this GitHub Archive homepage. Maybe we can catch up some time and riff on some ideas.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -169,7 +194,7 @@ <h2 id="resources"><a href="#resources">Research, visualizations, talks...</a></
             <li>
             <a href="http://datahackermd.com/2013/language-use-on-github/">Language Use on GitHub</a> visualization of the rank correlation between languages.</li>
             <li><a href="http://www.fastcolabs.com/3015178/the-top-10-hottest-github-projects-right-now?partner=rss">The Top 11 Hottest GitHub Projects Right Now</a></li>
-            <li>"New repos &amp; most watched" <a href="https://github.com/hnq90/GitHubArchive">email reports</a> (<a href="http://us6.campaign-archive1.com/?u=ecbe15df98da933fc8209cf1b&id=31fc9cb07c">example</a>). Signup: <a href="http://eepurl.com/bb4EFL">daily</a>, <a href="http://eepurl.com/bc5JV9">every 3 days</a>, <a href="http://eepurl.com/bc5Yn9">weekly</a>.</li>
+            <li><a href="http://thechangelog.com/nightly/">Subscribe to Changelog Nightly</a> (the new and improved GitHub Archive daily email reports). It ships every night at 10pm CT &mdash; and unearths the hottest new repos on GitHub before they blow up. It's nerd to the core and in your inbox each night.</li>
```
Comment (igrigorik) [inline index.html]: Let's keep this one as is, albeit we should probably add a note indicating that its a set of alternative reports.. Want to give credit where its due :)

**Comment by igrigorik** [comment]
Overall: :+1: ... The preview image is kinda large, but we can revisit that later :)

File: index.html (hunk 1 above)
Comment (adamstac) [inline index.html]: Word, I'll add it back. My concern was confusion.

**Comment by adamstac** [comment]
I can reduce the image size. It's it the weight or the height that's your concern?

**Comment by igrigorik** [comment]
@adamstac the height, it seems to dominate the page :)

**Comment by adamstac** [comment]
What do you think about this version?

changelog-nightly-preview

**Comment by igrigorik** [comment]
@adamstac apologies about the delay! I simplified things a bit and split the content between different sections. Should be live now.

**Comment by adamstac** [comment]
No worries, thanks for merging. Changelog Nightly is going/growing steady without a hitch.

**Comment by igrigorik** [comment]
\o/

**Pull request merged**